src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java

```diff
@@ -44,17 +49,77 @@
 
     private static final String ENTITY = "test";
 
-    private final LightblueClient client;
+    private static final String CLIENT_ERROR_DELIMITER = " , ";
```

Comment:
Nevermind, I see this is used for printing the error message and not parsing lightblue-clients.config. We're good.